setup.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from setuptools import setup, find_packages
+
+
+setup(
+    name='FoxPuppet',
+    version='0.1.0',
+    description='Foxpuppet for Firefox UI testing',
+    long_description='See http://foxpuppet.readthedocs.org/',
```

Comment:
We should add a README and use that for the `long_description`. See https://github.com/pytest-dev/pytest-selenium/blob/master/setup.py#L6 for an example.